src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -32,6 +35,25 @@ public OverseasEntitySubmissionDtoValidator(EntityNameValidator entityNameValida
 
     public Errors validateFull(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
 
+        if (isRoeUpdateEnabled && StringUtils.isNotBlank(overseasEntitySubmissionDto.getEntityNumber())) {
+            validateUpdateDetails(overseasEntitySubmissionDto, errors, loggingContext);
+        } else {
+            validateRegistrationDetails(overseasEntitySubmissionDto, errors, loggingContext);
+        }
+        return errors;
+    }
+
+    private void validateUpdateDetails(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
```

Comment:
i'd include the word Full in the methods to distinguish between full and partial validation rules, so the methods 
validateUpdateDetails -> validateFullUpdateDetails
and
validateRegistrationDetails -> validateFullRegistrationDetails